HelpSource/Classes/Process.schelp

```diff
@@ -56,3 +58,232 @@ list::
+'fileMain.scd tasks finished'.postln;
+)
+::
+## Create an SCD file called fileForLoad.scd in your home folder using the following block of code:
+code::
+("	in fileForLoad.scd:                    " + thisProcess.nowExecutingPath).postln;
+(
+testFunction: {
+	("	in the testFunction in fileForLoad.scd:" + thisProcess.nowExecutingPath).postln
+}
+)
+::
+::
+
+## Execute SCD file:
+
+list::
+## In SC-IDE or other editors:
+
+numberedlist::
+## Create a new SCD file (you do not need to save it for this step), then evaluate the following code:
+
+code::
+"~/fileMain.scd".standardizePath.openOS;
+// Note:
+// - .openDocument only works in SC-IDE.
+// - .openOS will work in other editors and
+//   the system default application will open the SCD file.
+::
+
+## Post window returns:
+
+teletype::
+-> /Users/_Your_Account_Folder_/fileMain.scd
+thisProcess.nowExecutingPath:
+	in fileMain.scd:                        /Users/_Your_Account_Folder_/fileMain.scd
+	in fileForLoad.scd:                     /Users/_Your_Account_Folder_/fileForLoad.scd
+	in the testFunction in fileForLoad.scd: /Users/_Your_Account_Folder_/fileMain.scd
+fileMain.scd tasks finished
+-> fileMain.scd tasks finished
+::
+::
+
+## In command line interface (CLI):
+
+code::
+// Linux:
+sclang '/home/_Your_Account_Folder_/fileMain.scd'
+
+// macOS (/Applications/SuperCollider.app might be changed to properly):
+/Applications/SuperCollider.app/Contents/MacOS/sclang ~/fileMain.scd
+
+// Windows (C:\Program Files\SuperCollider_dev\ should be changed to properly):
+"C:\Program Files\SuperCollider_dev\sclang.exe" %userprofile%\fileMain.scd
+::
+
+sclang returns to the terminal window (command prompt window on Windows):
+
+teletype::
+... (messages are truncated) ...
+*** Welcome to SuperCollider 3.13.0. *** For help press F1.
+thisProcess.nowExecutingPath:
+        in fileMain.scd:                        C:\Users\_Your_Account_Folder_\fileMain.scd
+        in fileForLoad.scd:                     C:\Users\_Your_Account_Folder_\fileForLoad.scd
+        in the testFunction in fileForLoad.scd: C:\Users\_Your_Account_Folder_\fileMain.scd
+fileMain.scd tasks finished
+::
+::
+::
+::
+::
+
+strong::Example 2.:: link::Classes/CmdPeriod::, link::Classes/ServerBoot:: and link::Classes/ServerTree:::
+
+table::
+##
+warning::The retuned path depends on the existence of the startup.scd file when the interpreter boots.::
+
+When used in link::Classes/CmdPeriod::, link::Classes/ServerBoot:: and link::Classes/ServerTree::, it will return teletype::nil:: unless startup.scd exists.
+
+code::
+(
+CmdPeriod.add {
+	var path = thisProcess.nowExecutingPath;
+	("- CmdPeriod’s thisProcess.nowExecutingPath:" + path).postln
+};
+ServerBoot.add {
+	var path = thisProcess.nowExecutingPath;
+	("- ServerBoot’s thisProcess.nowExecutingPath:" + path).postln
+};
+ServerTree.add {
+	var path = thisProcess.nowExecutingPath;
+	("- ServerTree’s thisProcess.nowExecutingPath:" + path).postln
+};
+s.reboot
+)
+::
```

Comment:
maybe add a note how to remove them again?

Reply:
ok I will do!